samples/client/petstore/python-experimental/tests/test_api_client.py

```diff
@@ -41,20 +43,32 @@ def test_configuration(self):
         client = petstore_api.ApiClient(config)
 
         # test prefix
-        self.assertEqual('PREFIX', client.configuration.api_key_prefix['api_key'])
+        self.assertEqual('PREFIX', client.configuration.api_key['api_key']['prefix'])
```

Comment:
@sebastien-rosset optionally include "=" or " " at the end of 
config.api_key['api_key']["prefix"]

Reply:
Do you mean something like this?
```
config.api_key['api_key'] = {'value': '123456', 'prefix': 'PREFIX '}
```
instead of:
```
config.api_key['api_key'] = {'value': '123456', 'prefix': 'PREFIX'}
```

Reply:
Yes, something like that. Again this is just a nice to have not required. Your code properly exercises the feature and shows that it is working.

Reply:
I made the change.